src/components/language/language.service.ts

```diff
@@ -403,33 +490,97 @@ export class LanguageService {
     _input: ProjectListInput,
     _session: ISession
   ): Promise<ProjectListOutput> {
-    // TODO
+    const result = await this.db
+      .query()
+      .matchNode('language', 'Language', { id: _language.id, active: true })
+      .match([
+        node('language'),
+        relation('out', '', 'project', { active: true }),
+        node('project', 'Project', {
+          active: true,
+        }),
+      ])
+      .return({
+        project: [{ id: 'id' }],
+      })
+      .run();
+
+    this.logger.info(`Language project lists ${result.length}`);
```

Comment:
Don't inject dynamic data into log messages. It makes it harder to track them in production.
```suggestion
    this.logger.debug(`list projects results`, { total: results.length });
```